components/Styling/StyledForm.js

```diff
@@ -0,0 +1,53 @@
+import styled from "styled-components";
+
+export const FormContainer = styled.form`
+  width: 95%;
+  margin: 20px;
+  padding: 20px 0px;
+  background-color: ${(props) => `var(--${props.backgroundcolor})`};
+  // background-color: var(--globalPlaceBackgroundColor);
+  border-radius: var(--globalborderradius);
```

Comment:
As you used camelCase for naming your css variables you should use this naming convention for all your css variables.